docs/ref/contrib/gis/db-api.txt

```diff
@@ -406,10 +406,10 @@ Function                              PostGIS  Oracle         MariaDB      MySQL
 :class:`ForcePolygonCW`               X                                                X
 :class:`FromWKB`                      X        X              X            X           X
 :class:`FromWKT`                      X        X              X            X           X
-:class:`GeoHash`                      X                                    X           X (LWGEOM/RTTOPO)
+:class:`GeoHash`                      X                       X (≥ 11.7)   X           X (LWGEOM/RTTOPO)
 :class:`Intersection`                 X        X              X            X           X
 :class:`IsEmpty`                      X
-:class:`IsValid`                      X        X                           X           X
+:class:`IsValid`                      X        X              X (≥ 11.7)   X           X
```

Comment:
I was just checking the compatibility tables, are we able to support the `isvalid` lookup for MariaDB ≥ 11.7 also?

Reply:
Yes

Reply:
I will update PR later today.

Reply:
Docs updated, great catch :dart: